programs/disks/DisksApp.cpp

```diff
@@ -2,6 +2,8 @@
 #include <Client/ClientBase.h>
 #include <Client/ReplxxLineReader.h>
 #include "Common/Exception.h"
+#include "Common/Logger.h"
```

Comment:
`"` for non-generated files makes me sad. Not a big deal

Reply:
This is likely old `clangd`, once you will upgrade to clang 20, everything will work like a charm (thanks to https://github.com/llvm/llvm-project/pull/67749)